### DISCUSSION TITLE
Is an adapter necessary for packages?

### DISCUSSION THREAD

**Discussion by vekunz** [sveltejs/kit#8195] [Q&A] [answered]
Hi, we have two package projects. Currently, we have `@sveltejs/adapter-auto` configured in our package projects. But actually, I do not see any reason for any adapter to be configured in a package project. Is an adapter really necessary for a package project, or can I just drop the adapter in the configuration?

**Comment by mavdotj** [comment, marked as answer]
An adapter is not required for packages, only if you are making documentation for your library you will have to include an adapter to build your site.
